src/CasparCG.ts

```diff
@@ -1664,8 +1664,8 @@ export class CasparCG extends EventEmitter implements ICasparCGConnection, Conne
 	/**
 	 * <http://casparcg.com/wiki/CasparCG_2.1_AMCP_Protocol#THUMBNAIL_LIST>
 	 */
-	public thumbnailList(): Promise<IAMCPCommand> {
-		return this.do(new AMCP.ThumbnailListCommand())
+	public thumbnailList(subFolder: string): Promise<IAMCPCommand> {
```

Comment:
Can you make this argument an optional argument?
Otherwise code looks fine to me, and we could merge this. ping @jesperstarkar

Reply:
I´ll do that :)